config/bootstrap.php

```diff
@@ -22,6 +22,13 @@
     class_alias('Cake\Utility\Text', 'Cake\Utility\String');
 }
 
+// @deprecated Backward compatibility with 2.x, 3.0.x
+class_alias('Cake\Mailer\AbstractTransport', 'Cake\Network\Email\AbstractTransport');
+class_alias('Cake\Mailer\Transport\DebugTransport', 'Cake\Network\Email\DebugTransport');
+class_alias('Cake\Mailer\Email', 'Cake\Network\Email\Email');
+class_alias('Cake\Mailer\Transport\MailTransport', 'Cake\Network\Email\MailTransport');
+class_alias('Cake\Mailer\Transport\SmtpTransport', 'Cake\Network\Email\SmtpTransport');
```

Comment:
Moving this to app-template serves no purpose if we make the app-template require correct cakephp version which then does not need any aliassing.
The aliassing is needed in core for BC to applications using the current classnames.